test/modules/helpers.spec.ts

```
@@ -1150,6 +1150,29 @@ describe('helpers', () => {
         it('should not trim whitespace', () => {
           expect(faker.helpers.fake('   ---   ')).toBe('   ---   ');
         });
+
+        it('should be able to handle methods that return a complex object', () => {
+          expect(
+            faker.definitions.airline.airport.map((airport) => airport.iataCode)
+          ).toContain(faker.helpers.fake('{{airline.airport.iataCode}}'));
+        });
+
+        it('should return undefined if a method returns a complex object but the property is undefined', () => {
+          expect(faker.helpers.fake('{{airline.airport.code}}')).toBe(
```

Currently if a property is unknown on we receive an error: 

```md
Invalid module method or definition: person.abc
- faker.person.abc is not a function
- faker.definitions.person.abc is not an array
``` 

I would prefer if we could throw this error here as well and add a case for `faker.person is a function but does not return an object with the property abc`. I'll gladly take suggestions on the phrasing.